### ISSUE TITLE
.deb package out of date

### ISSUE THREAD

**Issue by andy-ms** [Microsoft/vscode#50300]
Issue Type: <b>Bug</b>

I'm running vscode on linux mint, and I use the update manager to update vscode. Lately I haven't seen any updates in the update manager, but every time I open a window with code-insiders, there is a popup telling me that there is an update available -- clicking that takes me to a download page, but I'd rather get updates through my OS' update manager than having to do a manual install every time there's an update (which for code-insiders should be often).

This issue is basically a duplicate of #45868 but I don't see a way to comment on that.

VS Code version: Code - Insiders 1.24.0-insider (0d376d890ceb406cabd2b4281deebf5f4038a78c, 2018-05-15T05:11:37.091Z)
OS version: Linux x64 4.13.0-37-generic

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-6700HQ CPU @ 2.60GHz (8 x 2600)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>rasterization: disabled_software<br>video_decode: unavailable_software<br>video_encode: enabled<br>vpx_decode: unavailable_software<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|4, 5, 3|
|Memory (System)|7.63GB (1.12GB free)|
|Process Argv|/usr/share/code-insiders/code-insiders src|
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (1)</summary>

Extension|Author (truncated)|Version
---|---|---
typescript-javascript-grammar|ms-|0.0.41

</details>

**Comment by Tyriar** [comment]
When you update your OS, at least for Ubuntu, apt sources will be commented out. Maybe that's what happened? Can you check `/etc/apt/sources.list.d/vscode.list`?

**Comment by andy-ms** [comment]
That file contains `deb [arch=amd64] https://packages.microsoft.com/repos/vscode stable main`.

**Comment by andy-ms** [comment]
In the meanwhile, is there a way to disable the popup?

**Comment by Tyriar** [comment]
No way currently (see https://github.com/Microsoft/vscode/issues/21651). If you have the repo installed correctly then we could have just released another insiders build that day which the repo does not get

**Comment by andy-ms** [comment]
@Tyriar The deb source does not seem to be commented out but I haven't gotten any updates this week, doesn't that indicate some problem?

**Comment by Tyriar** [comment]
You're right there hasn't been an insiders for a while https://packages.microsoft.com/repos/vscode/pool/main/c/code-insiders/

I looked into it, I think the build has been red in the nightly run for over a week. I'll set a reminder to do a manual release when I'm on campus tomorrow.

**Comment by andy-ms** [comment]
Thanks!

**Comment by gusbemacbe** [comment]
I also have this problem on [K/X]Ubuntu. I added the GPG and PPA. 

I received update alerts of VSCode Insiders and I solved to run `sudo apt update` but code and code-insiders are not listed in `apt list --upgradable`. I solved to run `sudo apt install code-insiders`, but the terminal tells it is already up-to-date.

Here is the preview of `vscode.list`:

```shell
deb [arch=amd64] https://packages.microsoft.com/repos/vscode stable main
```
